src/mixins/intermediateMailIcon.js

```diff
@@ -1,9 +1,7 @@
-import intermediateMailSymbol from '@/assets/intermediate-mail.svg';
-
-export default function getIntermediateMailIconShapeAttributes(stroke = '#000', strokeWidth = 1, fill = '#fff') {
+export default function getIntermediateMailIconShapeAttributes(stroke = '#000', strokeWidth = 1, fill = '#fff', icon) {
```

Comment:
I think this is problematic. You've added a new argument that doesn't have a default value after arguments that do. This forces all callers of this function to supply arguments for all values. You should either move the `icon` argument to be first (PHP/WebStorm will do this refactoring for you) or remove the default values from this function.